p4rmyknife/modules/config.py

```diff
@@ -16,3 +16,13 @@ def install_client(args):
        p4_url = 'https://louis-test-p4k.s3.us-east-2.amazonaws.com/TestFile.sh'
       #urllib.request.urlretrieve(p4_url, 'p4v.tgz')
        urllib.request.urlretrieve(p4_url, 'TestFile.sh')
+
+    #Need to write script that will check for the downloaded file- 
+    #Testfile.sh, then run it, if present, or produce error if not present.
+    
+def check_file(args):
+try:
```

Comment:
Indentation is off here.  This isn't actually "part" of the function since it isn't indented.

Reply:
The indentation and white space concept is definitely something I need to get used to. I have been reading that it DOES matter for python, but I need to learn the details behind it. It's still good to know that space/indentation is integral for the code to work properly.